proposals/XXXX-compact.md

```diff
@@ -0,0 +1,280 @@
+---
+layout: default
+title: The Compact Programming Language
+parent: Hyperledger Improvement Proposals
+---
+
+# HIP Identifier
+<mark>**HIP identifier** a short description plus a serial number with a
+version.
+</mark>
+
+# Sponsor(s)
```

Comment:
Are there any sponsors outside of Shielded? See [Project Incubation Entry Considerations](https://lf-decentralized-trust.github.io/governance/guidelines/project-incubation-entry-considerations.html#sponsors) for sponsorship expectations and who might be a sponsor.